src/Summoner/CLI.hs

```diff
@@ -1,7 +1,8 @@
-{-# LANGUAGE ApplicativeDo   #-}
-{-# LANGUAGE QuasiQuotes     #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TupleSections   #-}
+{-# LANGUAGE ApplicativeDo       #-}
+{-# LANGUAGE QuasiQuotes         #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Can you add `ScopedTypeVariables` to the default extensions in the `.cabal` file. And also `TypeApplications`. It's very strange we don't have it in there..

Reply:
I added `TypeApplications` to `.cabal` but it didn`t help.